src/main/java/org/datacleaner/extension/xml/XPathTransformer.java

```diff
@@ -57,7 +57,7 @@
     private TransformerFactory transformerFactory;
 
     @Initialize
-    public void init() {
+    public void init() throws IllegalArgumentException {
```

Comment:
It's not super good practice to add runtime exception to throws, as it hides what the a user of the class actually needs to explicitly catch.

If you want to document it,  better add it in JavaDoc using @throws